etc/workflows/duplicate-event.xml

```diff
@@ -59,6 +59,14 @@
       </configurations>
     </operation>
 
+    <operation
+      id="start-workflow">
+      <configurations>
+        <configuration key="media-packages">${duplicate_media_package_ids}</configuration>
+        <configuration key="workflow-definition">fast</configuration>
```

Comment:
The WOH itself is cool, but I don't agree with this change to the workflow definition. We shouldn't include fast, since it's a testing workflow and people shouldn't run this by accident on their production cluster (as just happened to us), and also - why include an import workflow here? Since the initial event that was duplicated was already processed, the import has already happened, the only thing we might want to do is publish - but I wouldn't do that by default either.